python/ray/util/client/common.py

```diff
@@ -75,9 +77,12 @@
     ("grpc.http2.min_ping_interval_without_data_ms", GRPC_KEEPALIVE_TIME_MS - 50),
     # Allow many strikes
     ("grpc.http2.max_ping_strikes", 0),
+    (
+        "grpc.max_concurrent_streams",
+        int(os.getenv("RAY_CLIENT_SERVER_MAX_CONNECTIONS", CLIENT_SERVER_MAX_THREADS)),
```

Comment:
```suggestion
        int(CLIENT_SERVER_MAX_THREADS),
```

Reply:
I'm not sure whether we should use the same OS envs for this or not. I mean they are for different purpose I guess.
Do you mind telling me why do you think we should make it equal with that?

Reply:
Oh sorry, it should be half of that right? https://github.com/ray-project/ray/blob/master/python/ray/util/client/server/dataservicer.py#L368

Reply:
Hmmm, I probably doesn't even need this to fix the issue.